bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/execution/ProcessExecutorImpl.java

```diff
@@ -163,15 +164,15 @@
     private final RefBusinessDataService refBusinessDataService;
 
     public ProcessExecutorImpl(final ActivityInstanceService activityInstanceService, final ProcessInstanceService processInstanceService,
-                               final TechnicalLoggerService logger, final FlowNodeExecutor flowNodeExecutor, final WorkService workService,
-                               final ProcessDefinitionService processDefinitionService, final GatewayInstanceService gatewayInstanceService,
-                               final TransitionService transitionService, final EventInstanceService eventInstanceService, final ConnectorService connectorService,
-                               final ConnectorInstanceService connectorInstanceService, final ClassLoaderService classLoaderService, final OperationService operationService,
+            final TechnicalLoggerService logger, final FlowNodeExecutor flowNodeExecutor, final WorkService workService,
```

Comment:
a lot of formartting, sometimes the PR add spaces here sometimes it removes some. Which one is correct?